### PR TITLE
[leapfrog] public pages: add instagram to share options

### PR DESCRIPTION
**Pull request by matallo**
inspired by https://spotify-foundthemfirst.com/, added an option to share in instagram (user will have to save the card in the gallery, then share it in Instagram)

screen shot 2015-09-11 at 18 15 05

CR @xavijam @saleiva

### CODE REVIEW

File: app/assets/stylesheets/common/icon-font.css.scss [hunk 1]
```diff
@@ -368,4 +368,8 @@
 
 .iconFont-Table:before {
   content: "\E058";
+}
+
+.iconFont-Instagram:before {
```
Comment (saleiva) [inline app/assets/stylesheets/common/icon-font.css.scss]: How the result looks like? Can you upload a gif?

Comment (matallo) [inline app/assets/stylesheets/common/icon-font.css.scss]: not that I know, in any case we're not generating gifs yet for the public pages

Comment (saleiva) [inline app/assets/stylesheets/common/icon-font.css.scss]: I mean for showing the workflow

**Comment by saleiva** [comment]
bump!

**Comment by xavijam** [comment]
Hey, what about this? I'd like to see the results :).

**Comment by matallo** [comment]
screenshot_2015-11-27-18-49-12
screenshot_2015-11-27-18-49-34
screenshot_2015-11-27-18-49-39
screenshot_2015-11-27-18-49-43
screenshot_2015-11-27-18-49-48
screenshot_2015-11-27-18-50-25

sorry I've been very lazy to update branch, deploy to staging and take the captures, it's in ded05

**Comment by xavijam** [comment]
Don't set a review of a pull request for a whole group, please.

**Comment by matallo** [comment]
sure, this is a product decision, too so happy to know @saleiva opinion here (it's true it is not the most straight forward interaction)

**Pull request closed without merging**